composer.json

```diff
@@ -29,7 +29,7 @@
         "friendsofsymfony/http-cache": "^2.1.3",
         "friendsofsymfony/http-cache-bundle": "~2.0",
         "friendsofsymfony/jsrouting-bundle": "^2.3",
-        "friendsofsymfony/rest-bundle": "^2.2",
+        "friendsofsymfony/rest-bundle": "^2.4",
```

Comment:
upgrade is needed else the fos rest bundle does not work correctly: https://travis-ci.org/sulu/sulu/jobs/583820505